apidoc/Titanium/UI/BlurView.yml

```
@@ -0,0 +1,176 @@
+---
+name: Titanium.UI.BlurView
+summary: |
+  A blur view providing a background blur effect similar to iOS' UIVisualEffectView.
+
+  The blur is applied to content behind the blur view within the window. Place it above
+  other views to blur anything visible underneath within its bounds.
+extends: Titanium.UI.View
+platforms: [android, iphone, ipad, macos]
+since: { android: "13.0.0" }
+properties:
+
+  - name: effect
+    summary: The blur effect to apply to the effect view.
+    description: |
+        On iOS, you can also use the Titanium.UI.iOS.BLUR_EFFECT_STYLE_* effects
+        that are platform-specific.
```

```suggestion
        On iOS, you can also use the Titanium.UI.iOS.BLUR_EFFECT_STYLE_* effects
        that are platform-specific.
        On Android it will add a default blurRadius of 16px. If you want to change the value you have to use `blurRadius` and `overlayColor` instead of the `effect` preset.
```

That took me a bit to understand that I can't change the blurRadius in an `effect` preset. So I guess it should be in the docs.